mobile/lib/common/loading_screen.dart

```diff
@@ -19,11 +23,27 @@ class _LoadingScreenState extends State<LoadingScreen> {
   @override
   void initState() {
     super.initState();
-    Future.wait<dynamic>(
-            [Preferences.instance.hasEmailAddress(), Preferences.instance.getOpenPosition()])
-        .then((value) {
+    Future.wait<dynamic>([
+      Preferences.instance.hasEmailAddress(),
+      Preferences.instance.getOpenPosition(),
+      isSeedFilePresent(),
+    ]).then((value) {
       final hasEmailAddress = value[0];
       final position = value[1];
+      final isSeedFilePresent = value[2];
+
+      logger.d("Scanning for seed file: $isSeedFilePresent");
+
+      if (isSeedFilePresent) {
+        runBackend(context).then((value) {
+          logger.i("Backend started");
+        });
+      } else {
+        // No seed file: let the user choose whether they want to create a new
+        // wallet or import their old one
+        GoRouter.of(context).go(WalletChoiceScreen.route);
+        return;
+      }
```

Comment:
I would let the user enter the email first and then check for the seed file. 
@holzeis / @luckysori : what do you think?

Reply:
I do agree, but a little bit more effort as we can only complete the registration after the node id is known.